### ISSUE TITLE
Diagonal Joystick Mapping and Mouse Wheel Inputs

### ISSUE THREAD

**Issue by Piximator** [magicien/JoyKeyMapper#2]
There's only a couple of problems with this software as far as I know, and one of them is that there's no way to set the joystick to a certain mapping when it's moved diagonally. For example, I may want the left joystick to press the keys W and A at the same time when I move it to the top-right corner.

The only other issue I encountered was that there was no way to set a button to scroll the mouse wheel by a certain increment. This would be useful in a game like Minecraft where I can map the L and R buttons to move the scroll wheel to select an item in my inventory.

Other than that, there are no other problems. Great job with this software!

**Comment by FonkinS** [comment]
Yes, please do fix the diagonals, incredibly annoying for Twin Stick shooter like games
